assets/js/analyzer.js

```diff
@@ -56,7 +61,7 @@ function getAnalyzers() {
             deferred.resolve();
         }
         else {
-            console.warn('Analyzers cache ' + (analyzers === null ? 'stale' : 'miss') + ', retrieving from server');
+            console.warn('Analyzers cache ' + (analyzers ? 'miss' : 'stale') + ', retrieving from server');
```

Comment:
Seems backwards.

Reply:
The original ternary was true for null values.
The new ternary is true for non-null values.
Also, I flipped the respective values.
Should be fine.